Demo/src/main/java/com/paypal/android/ui/DemoApp.kt

```diff
@@ -98,6 +99,9 @@ fun DemoApp() {
                 composable(DemoAppDestinations.PAYPAL_WEB) {
                     PayPalWebView()
                 }
+                composable(DemoAppDestinations.PAYPAL_WEB_VAULT) {
+                    PayPalWebVaultView()
```

Comment:
For now it's simplicity. Each composable also typically has an associated `<FEATURE_NAME>ViewModel` type. We could make the code more DRY, but it would require us to define a common `interface` between the two view models.

Reply:
Same on iOS unless we are talking about something different.